en-2016-07-17-latex-template-for-responses-to-referees-reports/statement.tex

```diff
@@ -1,63 +1,64 @@
-%
 % Title:    A LaTeX Template For Responses To a Referees' Reports
 % Author:   Petr Zemek <s3rvac@gmail.com>
 % Homepage: https://blog.petrzemek.net/2016/07/17/latex-template-for-responses-to-referees-reports/
 % License:  CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
-%
-\documentclass[a4paper,10pt]{article}
+\documentclass[10pt]{article}
 
-% Packages
+% Allow Unicode input (alternatively, you can use XeLaTeX or LuaLaTeX)
 \usepackage[utf8]{inputenc}
 
-% View
-\setlength{\hoffset}{-1.5cm}
-\setlength{\voffset}{-1.5cm}
-\setlength{\textheight}{23.0cm}
-\setlength{\textwidth}{15.2cm}
```

Comment:
Don't set these values yourself! Use the `geometry` package to make the necessary changes if you feel they need to be made.

I left out these changes entirely in my commit, but that's just personal preference.